### PR TITLE
Add NestJS tools libraries related to file storage and AMQP trasnport

### PR DESCRIPTION
**Pull request by getlarge**
## Resource type _(required)_

- [x] GitHub Repository:
    - [x] **(Required)** I confirm that the GitHub repository has more than **10 stars**.
    - [x] **(Required)** The repository is not archived.
- [ ] Video.
- [ ] Tutorial.
- [ ] Meetups.
- [ ] Improve documentation _(grammatical corrections, improve doc style, ...)_.

> If your contribution is NOT a GitHub repository, then you can skip the next titles, except "Rules".

## Description _(required)_

The `NestJS tools` contain libraries written over the past 4 years to provide reusable solutions for applications running in production.
I included three of those libraries, which I think are the most helpful and seemed to be missing during my research for equivalent.

## Link _(required)_

https://github.com/getlarge/nestjs-tools

## Rules _(required)_

- [x] **NestJS / Nest**: It's not nest, nestjs, nest.js, and other variants.
- [x] **Node.js**: It's not nodejs, node, and other variants.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -403,6 +405,8 @@
 #### File Storage
 
 - ![](https://img.shields.io/github/stars/codebrewlab/nestjs-storage.svg?style=flat-square) [`@codebrew/nestjs-storage`](https://github.com/codebrewlab/nestjs-storage) - A manage file storage module([flydrive](https://github.com/Slynova-Org/flydrive)) for NestJS Framework.
+- ![](https://img.shields.io/github/stars/getlarge/nestjs-tools.svg?style=flat-square) [@getlarge/nestjs-tools-file-storages](https://github.com/getlarge/nestjs-tools/blob/main/packages/file-storage) - A FileStorage Module supporting several strategies (**FS**, **S3**, **GCP storage**) to store files.
+- ![](https://img.shields.io/github/stars/getlarge/nestjs-tools.svg?style=flat-square) [@getlarge/nestjs-tools-fastify-upload](https://github.com/getlarge/nestjs-tools/tree/main/packages/fastify-upload) - Suite of Interceptors and request decorators to handle file upload with **Fastify**.
```
Comment (micalevisk) [inline README.md:409]:
```suggestion
- ![](https://img.shields.io/github/stars/getlarge/nestjs-tools.svg?style=flat-square) [`@getlarge/nestjs-tools-file-storages`](https://github.com/getlarge/nestjs-tools/blob/main/packages/file-storage) - A FileStorage Module supporting several strategies (**FS**, **S3**, **GCP storage**) to store files.
- ![](https://img.shields.io/github/stars/getlarge/nestjs-tools.svg?style=flat-square) [`@getlarge/nestjs-tools-fastify-upload`](https://github.com/getlarge/nestjs-tools/tree/main/packages/fastify-upload) - Suite of Interceptors and request decorators to handle file upload with **Fastify**.
```

File: README.md [hunk 2]
```diff
@@ -367,6 +368,7 @@
 - ![](https://img.shields.io/github/stars/nestjsx/nestjs-amqp.svg?style=flat-square) [`nestjs-amqp`](https://github.com/nestjsx/nestjs-amqp) - An amqp connection manager.
 - ![](https://img.shields.io/github/stars/AlariCode/nestjs-rmq.svg?style=flat-square) [`nestjs-rmq`](https://github.com/AlariCode/nestjs-rmq) - A custom library for NestJS microservice. It allows you to use RabbitMQ or AMQP.
 - ![](https://img.shields.io/github/stars/golevelup/nestjs.svg?style=flat-square) [`@golevelup/nestjs-rabbitmq`](https://github.com/golevelup/nestjs/tree/master/packages/rabbitmq) - Flexible AMQP integrations for NestJS that supports multiple messaging patterns and intuitive decorators.
+- ![](https://img.shields.io/github/stars/getlarge/nestjs-tools.svg?style=flat-square) [@getlarge/nestjs-tools-amqp-transport](https://github.com/getlarge/nestjs-tools/tree/main/packages/amqp-transport) - An extended **AMQP** transport strategy supporting **exchanges**.
```
Comment (micalevisk) [inline README.md:371]:
```suggestion
- ![](https://img.shields.io/github/stars/getlarge/nestjs-tools.svg?style=flat-square) [`@getlarge/nestjs-tools-amqp-transport`](https://github.com/getlarge/nestjs-tools/tree/main/packages/amqp-transport) - An extended **AMQP** transport strategy supporting **exchanges**.
```

**Review by micalevisk — CHANGES_REQUESTED** [review]
fix formatting

File: README.md (hunk 2 above)
Comment (getlarge) [reply README.md]: My eyes are bad that day :). Fixed.

**Review by jmcdo29 — APPROVED** [review]

**Pull request merged**